.env.development

```diff
@@ -12,4 +12,5 @@ REACT_APP_NODE_3 = "https://bsc-dataseed.binance.org"
 
 REACT_APP_GRAPH_API_PROFILE = "https://api.thegraph.com/subgraphs/name/pancakeswap/profile"
 REACT_APP_GRAPH_API_PREDICTION = "https://api.thegraph.com/subgraphs/name/pancakeswap/prediction"
+REACT_APP_GRAPH_API_LOTTERY = "https://api.thegraph.com/subgraphs/name/pancakeswap/lottery"
```

Comment:
It should be in the env file only if the value change based on the node_env value

Reply:
Good point - I've made a PR moving all of these unchanging values out of the env variables https://github.com/pancakeswap/pancake-frontend/pull/1550

Reply:
Ah, this does get changed at the env level sometimes within Netlify - we've had a few occasions where we needed to switch to using the BSC graph. I'm gonna leave them in.